Component Plug-in (CP) Examples/mapField/mapField/v1/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel='stylesheet' href='./map.css'></link>
+</head>
+
+<body>
+  <div id='map-container'></div>
+  <script src='APPIAN_JS_SDK_URI'></script>
+  <script>
+    let mapState = {};
+    let currentLocation = '';
+    let newKey, currentKey, newLocation;
+    let mapsApiScriptId = 'google-maps-api-script';
+
+    // Handle a new user input location from SAIL.
+    Appian.Component.onNewValue(function (newValues) {
+      newKey = newValues.key;
+      newLocation = newValues.location;
+      if (newLocation && !(currentKey || newKey)) {
+        Appian.Component.setValidations('The API Key provided is empty. Please update the parameter "Google API Key" with a valid key.');
+      }
+      if (newKey && currentKey !== newKey) {
+        let oldScript = document.getElementById(mapsApiScriptId);
+        if (oldScript) {
+          oldScript.parentElement.removeChild(oldScript);
+          document.getElementById('map-container').innerHTML = ''
+          google = null;
+          mapState = {};
+        }
+
+        currentKey = newKey;
+        let s = document.createElement('script');
+        s.setAttribute('id', mapsApiScriptId)
+        s.setAttribute('src', 'https://maps.googleapis.com/maps/api/js?key=' + newKey + '&callback=initMap');
+        document.body.appendChild(s);
+      }
+      if (mapState.scriptLoaded && currentLocation !== newLocation) {
+        changeLocation(newLocation);
+      }
+    });
+
+    function initMap() {
+      Appian.Component.setValidations('');
+      mapState.scriptLoaded = true;
+      changeLocation(newLocation);
+    }
+
+    function gm_authFailure() {
+      Appian.Component.setValidations('The API Key provided is invalid. Please update the parameter "Google API Key" with a valid key.');
+    };
+
+    function changeLocation(newLocation) {
+      if (newLocation) {
+        currentLocation = newLocation;
+        // Obtain coordinates for user input location
+        mapState.geocoder = mapState.geocoder || new google.maps.Geocoder();
+        mapState.geocoder.geocode({ 'address': newLocation }, function (results, status) {
+          if (status === 'OK') {
+            Appian.Component.setValidations('');
+            let latLng = results[0].geometry.location;
+            if (mapState.map) {
+              mapState.map.setCenter(latLng);
+            } else {
+              renderMap(latLng);
+            }
+          } else if (status === 'ZERO_RESULTS') {
+            Appian.Component.setValidations(`No matching location was found for input: "${newLocation}". Valid input must be a country, city, full address, or anything the Google Geocoding API supports (https://developers.google.com/maps/documentation/geocoding/intro).`);
```

Comment:
This syntax isn't supported in IE11 so we can't use it for this example - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Whomp whomp. Good catch

Reply:
I appreciate the effort 😢